2024-01-16T09:28:34+01:00 CRIT (2): Zend\Mvc\Router\Exception\RuntimeException: Route with name "tracker" does not have child routes in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Router\Http\TreeRouteStack.php:322
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Url.php(99): Zend\Mvc\Router\Http\TreeRouteStack->assemble(Array, Array)
#1 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Redirect.php(48): Zend\Mvc\Controller\Plugin\Url->fromRoute('tracker/tracker...', Array, Array, false)
#2 G:\xampp\htdocs\sigtrace\module\Common\Authentication\src\Common\Authentication\Controller\AuthenticationController.php(78): Zend\Mvc\Controller\Plugin\Redirect->toRoute('tracker/tracker...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): Common\Authentication\Controller\AuthenticationController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:29:50+01:00 CRIT (2): Zend\Mvc\Router\Exception\RuntimeException: Route with name "tracker" does not have child routes in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Router\Http\TreeRouteStack.php:322
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Url.php(99): Zend\Mvc\Router\Http\TreeRouteStack->assemble(Array, Array)
#1 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Redirect.php(48): Zend\Mvc\Controller\Plugin\Url->fromRoute('tracker/tracker...', Array, Array, false)
#2 G:\xampp\htdocs\sigtrace\module\Common\Authentication\src\Common\Authentication\Controller\AuthenticationController.php(78): Zend\Mvc\Controller\Plugin\Redirect->toRoute('tracker/tracker...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): Common\Authentication\Controller\AuthenticationController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:30:02+01:00 CRIT (2): Zend\Mvc\Router\Exception\RuntimeException: Route with name "tracker" does not have child routes in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Router\Http\TreeRouteStack.php:322
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Url.php(99): Zend\Mvc\Router\Http\TreeRouteStack->assemble(Array, Array)
#1 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Redirect.php(48): Zend\Mvc\Controller\Plugin\Url->fromRoute('tracker/tracker...', Array, Array, false)
#2 G:\xampp\htdocs\sigtrace\module\Common\Authentication\src\Common\Authentication\Controller\AuthenticationController.php(78): Zend\Mvc\Controller\Plugin\Redirect->toRoute('tracker/tracker...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): Common\Authentication\Controller\AuthenticationController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:30:12+01:00 CRIT (2): Zend\Mvc\Router\Exception\RuntimeException: Route with name "tracker" does not have child routes in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Router\Http\TreeRouteStack.php:322
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Url.php(99): Zend\Mvc\Router\Http\TreeRouteStack->assemble(Array, Array)
#1 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Redirect.php(48): Zend\Mvc\Controller\Plugin\Url->fromRoute('tracker/tracker...', Array, Array, false)
#2 G:\xampp\htdocs\sigtrace\module\Common\Authentication\src\Common\Authentication\Controller\AuthenticationController.php(78): Zend\Mvc\Controller\Plugin\Redirect->toRoute('tracker/tracker...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): Common\Authentication\Controller\AuthenticationController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:30:20+01:00 CRIT (2): Zend\Mvc\Router\Exception\RuntimeException: Route with name "tracker" does not have child routes in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Router\Http\TreeRouteStack.php:322
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Url.php(99): Zend\Mvc\Router\Http\TreeRouteStack->assemble(Array, Array)
#1 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\Plugin\Redirect.php(48): Zend\Mvc\Controller\Plugin\Url->fromRoute('tracker/tracker...', Array, Array, false)
#2 G:\xampp\htdocs\sigtrace\module\Common\Authentication\src\Common\Authentication\Controller\AuthenticationController.php(78): Zend\Mvc\Controller\Plugin\Redirect->toRoute('tracker/tracker...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): Common\Authentication\Controller\AuthenticationController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:33:31+01:00 CRIT (2): PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'sigtrace.form_109_199' doesn't exist in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php:360
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php(360): PDO->query('select as_id, a...')
#1 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Model\Dashboard.php(179): Zend\Db\Adapter\Driver\Pdo\Connection->execute('select as_id, a...')
#2 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Controller\IndexController.php(160): SigTRACE\Dashboard\Model\Dashboard->getAllProductList(Array, ' and date(aerre...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): SigTRACE\Dashboard\Controller\IndexController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:39:19+01:00 CRIT (2): PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'sigtrace.form_109_199' doesn't exist in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php:360
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php(360): PDO->query('select as_id, a...')
#1 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Model\Dashboard.php(179): Zend\Db\Adapter\Driver\Pdo\Connection->execute('select as_id, a...')
#2 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Controller\IndexController.php(160): SigTRACE\Dashboard\Model\Dashboard->getAllProductList(Array, ' and date(aerre...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): SigTRACE\Dashboard\Controller\IndexController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:40:16+01:00 CRIT (2): PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'sigtrace.form_109_199' doesn't exist in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php:360
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php(360): PDO->query('select as_id, a...')
#1 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Model\Dashboard.php(179): Zend\Db\Adapter\Driver\Pdo\Connection->execute('select as_id, a...')
#2 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Controller\IndexController.php(160): SigTRACE\Dashboard\Model\Dashboard->getAllProductList(Array, ' and date(aerre...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): SigTRACE\Dashboard\Controller\IndexController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
2024-01-16T09:40:56+01:00 CRIT (2): PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'sigtrace.form_109_199' doesn't exist in G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php:360
Stack trace:
#0 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Db\Adapter\Driver\Pdo\Connection.php(360): PDO->query('select as_id, a...')
#1 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Model\Dashboard.php(179): Zend\Db\Adapter\Driver\Pdo\Connection->execute('select as_id, a...')
#2 G:\xampp\htdocs\sigtrace\module\SigTRACE\Dashboard\src\SigTRACE\Dashboard\Controller\IndexController.php(160): SigTRACE\Dashboard\Model\Dashboard->getAllProductList(Array, ' and date(aerre...')
#3 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractActionController.php(82): SigTRACE\Dashboard\Controller\IndexController->indexAction()
#4 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#5 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#6 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#7 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Controller\AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#8 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#9 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#10 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#11 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#12 G:\xampp\htdocs\sigtrace\vendor\zendframework\zendframework\library\Zend\Mvc\Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#13 G:\xampp\htdocs\sigtrace\public\index.php(28): Zend\Mvc\Application->run()
#14 {main}
